app/models/chapter.rb

```diff
@@ -22,8 +26,20 @@
 # rubocop:enable Metrics/LineLength
 
 class Chapter < ApplicationRecord
+  include Titleable
+
   belongs_to :manga
 
+  has_attached_file :thumbnail
+
   validates :manga, presence: true
   validates :number, presence: true
+  # validates :volume_number, presence: true
```

Comment:
do we want to validate this? i'm thinking sometimes we might want to add chapters when volumes aren't out yet.